docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
You can't realistically use `channel_name` for anything if it's prefixed with the guid. You would have to register for all and sniff if it's likes, or comments, or whatever.